src/transport/middleware.rs

```diff
@@ -107,11 +108,11 @@ where
         req.query_parameters
             .insert("pnsdk".into(), format!("{}/{}", SDK_ID, VERSION));
         req.query_parameters
-            .insert("uuid".into(), self.user_id.clone());
+            .insert("uuid".into(), self.user_id.to_string());
 
         if let Some(instance_id) = &self.instance_id {
             req.query_parameters
-                .insert("instanceid".into(), instance_id.clone());
+                .insert("instanceid".into(), instance_id.to_string());
```

Comment:
Isn't `to_string` just a `clone` in new clothes?

Reply:
this is for conversion so `into()` looks better.

also since this is in send() method.. this will be short lived.